libsrc/gamate/joy/gamate-stdjoy.s

```diff
@@ -79,6 +79,7 @@ COUNT:
 
 READJOY:
         lda     JOY_DATA
+	eor     #$ff
```

Comment:
- We don't want to have tab chars in the cc65 code base.
- The predominant style is that hex chars are uppercase.

I can surely fix those things myself but if you want to see your pull request merged as-is then you need to adjust them...

Reply:
Ok! I will fix!